docs/source/conf.py

```diff
@@ -57,6 +57,7 @@
     "pytest": ("https://docs.pytest.org/en/latest/", None),
     "coverage": ("https://coverage.readthedocs.io/en/latest/", None),
     "remote_pdb": ("https://python-remote-pdb.readthedocs.io/en/latest/", None),
+    "cocotb1.9": ("https://docs.cocotb.org/en/v1.9.2/", None),
```

Comment:
```suggestion
    "cocotb19": ("https://docs.cocotb.org/en/v1.9.2/", None),
```
This might make the links work - they vanish to the empty string right now.